quickstarts/jbpm-workitems-camel/src/main/java/org/jboss/integration/fuse/quickstarts/jbpm/workitems/camel/ApplicationServlet.java

```diff
@@ -0,0 +1,112 @@
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.jbpm.process.workitem.camel.CamelHandler;
+import org.jbpm.process.workitem.camel.CamelHandlerFactory;
+import org.kie.api.KieServices;
+import org.kie.api.runtime.KieContainer;
+import org.kie.api.runtime.KieSession;
+
+public class ApplicationServlet extends HttpServlet {
+
+    int APPLICATION_COUNT = 10; // The number of applications to create at the activation of bundle
+
+    /* Range for mortgage ammount */
+    int MORTGAGE_MIN = 100000;
+    int MORTGAGE_MAX = 1000000;
+
+    /* The range for applicant income */
+    int INCOME_MIN = 20000;
+    int INCOME_MAX = 200000;
+
+    KieSession kieSession;
+
+    File tempDir;
+
+    private static final String ACCEPTED_APPS_FILENAME = "acceptedApplications.txt";
+    private static final String REJECTED_APPS_FILENAME = "rejectedApplications.txt";
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+
+        tempDir = new File(System.getProperty("java.io.tmpdir"));
+
+        CamelHandler handler = CamelHandlerFactory.fileHandler();
+
+        KieServices kieServices = KieServices.Factory.get();
+        KieContainer kieContainer = kieServices.newKieClasspathContainer(this.getClass().getClassLoader());
+        kieSession = kieContainer.newKieSession("camel-workitem-ksession");
+        kieSession.getWorkItemManager().registerWorkItemHandler("CamelFile", handler);
+
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+        Random random = new Random();
+
+        /* Create 10 applications. Each application is evaluated by "camelFileProcess" */
+        for (int i = 1; i <= 10; i++) {
+            /* Create application */
+            MortgageApplication mortgageApplication = new MortgageApplication();
+            mortgageApplication.setId(i);
+            mortgageApplication.setIncome(random.nextInt(INCOME_MAX - INCOME_MIN) + INCOME_MIN);
+            mortgageApplication.setAmmount(random.nextInt(MORTGAGE_MAX - MORTGAGE_MIN) + MORTGAGE_MIN);
+
+            /* Initialize process parameters */
+            Map<String, Object> params = new HashMap<String, Object>();
+            params.put("payloadVar", "");
+            params.put("pathVar", tempDir.getAbsolutePath());
+            params.put("fileNameVar", "");
+            params.put("applicationVar", mortgageApplication);
+
+            kieSession.startProcess("camelFileProcess", params); // run process
+        }
+        
+        try {
+            Thread.sleep(5000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```

Comment:
I understand that this is just a quickstart, but using `e.printStackTrace()` does not seem ideal. Maybe we could log the exception?